ui/src/pages/plan/daybyday/DayByDayContainer.tsx

```diff
@@ -119,7 +120,9 @@ export const DayByDayContainer = ({ userid, currentTime }: { userid: string, cur
 
     if (loading) {
         return <div style={{ minHeight: '100%', width: '100%' }} className="text-center">
-            <h5 data-testid="daybyday-loading">Loading...</h5>
+            <div className="spinner-border" role="status">
+                <span data-testid="daybyday-loading" className="visually-hidden"></span>
```

Comment:
Looks like the black one: https://getbootstrap.com/docs/5.0/components/spinners/